### PR TITLE
add tools

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
Generations Archive Editor, GenerationsSkillEdit, PACTool, and Rail Canyon.

### CODE REVIEW

File: docs/tools/RSDK/common/files.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+[GitHub Repository :fontawesome-brands-github:](https://github.com/Rubberduckycooly/RetroED){ .md-button target="_blank"}
+
+# Animation Tools
+
+## RSDK
+RSDK reverse stuff + Sonic Mania Animation Editor.
+
+[Download :fontawesome-solid-download:](https://github.com/Xeeynamo/RSDK/releases){ .md-button .md-button--primary target="_blank"}
+[GitHub Repository :fontawesome-brands-github:](https://github.com/Xeeynamo/RSDK){ .md-button target="_blank"}
```
Comment (PTKickass) [inline docs/tools/RSDK/common/files.md:24]: These should be split into different files whenever possible. So you probably can split Model, File Format and Animation tools into separate archives

Comment (PTKay) [inline docs/tools/RSDK/common/files.md:24]: These should be split into different files. So, Model, File Format and Animation tools. If these file formats have a specific file type, then you can just create archives for said file types.

File: docs/tools/Renderware/common/files/txd.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+description: TXD tools for Renderware Sonic games
+---
+# TXD Tools
+
+## TXD_Hack
+TXD texture archive editor writen with meticulous documentation.
```
Comment (PTKay) [inline docs/tools/Renderware/common/files/txd.md:7]: "meticulous documentation" doesn't sound too great. You can simplify this description by just typing this:

```
TXD texture archive editor.
```

Also, isn't there a better way to edit texture for these games nowadays? This seems really old.

File: docs/tools/hedgehog-engine/blueblur/files/ar.md [hunk 3]
```diff
@@ -6,4 +6,9 @@ description: AR/ARL tools for Sonic Generations
...
 Tool to compress and decompress AR files for Sonic Generations, using 'makecab' and 'expand'.
 
-[Download :fontawesome-solid-download:](https://github.com/PTKickass/CompressAR/releases){ .md-button .md-button--primary target="_blank"}
\ No newline at end of file
+[Download :fontawesome-solid-download:](https://github.com/PTKickass/CompressAR/releases){ .md-button .md-button--primary target="_blank"}
+
+## Generations Archive Editor
+Editor for Sonic Generations .ar.00/.pfd archives.
+
+[GitHub Repository :fontawesome-brands-github:](https://github.com/sonicretro/Generations-Archive-Editor){ .md-button target="_blank"}
```
Comment (PTKay) [inline docs/tools/hedgehog-engine/blueblur/files/ar.md:14]: No need for this, you can remove it. Hedgehog Engine games already have a common archive editor.

File: docs/tools/hedgehog-engine/blueblur/files/sav.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+description: SAV tools for Sonic Generations
+---
+# SAV Tools
+
+## GenerationsSkillEdit
+Skill Set Editor for Sonic Generations save files.
+
+[GitHub Repository :fontawesome-brands-github:](https://github.com/sonicretro/GenerationsSkillEdit){ .md-button target="_blank"}
```
Comment (PTKay) [inline docs/tools/hedgehog-engine/blueblur/files/sav.md:9]: Save editors are not considered modding tools, so this file should be removed.

File: docs/tools/hedgehog-engine/common/lighting.md [hunk 5]
```diff
@@ -6,4 +6,9 @@ description: Lighting tools for the Hedgehog Engine
 ## HedgeGI
 Tool that allows you to bake global illumination and light field data for Hedgehog Engine games.
 
-[GitHub Repository :fontawesome-brands-github:](https://github.com/blueskythlikesclouds/HedgeGI){ .md-button .md-button--primary target="_blank"}
\ No newline at end of file
+[GitHub Repository :fontawesome-brands-github:](https://github.com/blueskythlikesclouds/HedgeGI){ .md-button .md-button--primary target="_blank"}
+
+## SonicLightTool
+Small collection of tools to read, write, and edit .light files for Sonic Unleashed/Generations/Lost World/Forces.
```
Comment (PTKay) [inline docs/tools/hedgehog-engine/common/lighting.md:12]: Please change this to
```
Small collection of tools to read, write, and edit .light files for Hedgehog Engine games.
```

File: docs/tools/renderware/common/levels.md [hunk 6]
```diff
@@ -7,4 +7,9 @@ description: Level editing tools for Renderware Sonic games
...
 
 [Download :fontawesome-solid-download:](https://github.com/igorseabra4/HeroesPowerPlant/releases){ .md-button .md-button--primary target="_blank"}
-[GitHub Repository :fontawesome-brands-github:](https://github.com/igorseabra4/HeroesPowerPlant){ .md-button target="_blank"}
\ No newline at end of file
+[GitHub Repository :fontawesome-brands-github:](https://github.com/igorseabra4/HeroesPowerPlant){ .md-button target="_blank"}
+
+## Rail Canyon
+Level viewer/editor for Sonic Heroes.
+
+[GitHub Repository :fontawesome-brands-github:](https://github.com/DeadlyFugu/rail-canyon){ .md-button target="_blank"}
```
Comment (PTKay) [inline docs/tools/renderware/common/levels.md:15]: Any difference between Rail Canyon and HeroesPowerPlant?

If yes, put Rail Canyon on a separate archive for heroes
If not, delete Rail Canyon.

File: mkdocs.yml [hunk 7]
```diff
@@ -15,6 +15,7 @@ nav:
           - XNCP: tools/hedgehog-engine/blueblur/files/xncp.md
           - GTE/GTM: tools/hedgehog-engine/blueblur/files/gte.md
           - AR/ARL: tools/hedgehog-engine/blueblur/files/ar.md
+          - SAV: tools/hedgehog-engine/blueblur/files/sav.md
```
Comment (PTKay) [inline mkdocs.yml:18]: Remove

**Review by PTKay — CHANGES_REQUESTED** [review]
Sorry for the late review. Anyways:

- RSDK should be renamed to retro-engine
- Is sonicRPG the codename for Sonic Adventure?
- There is no proof that Sonic and the Secret Rings uses Renderware, so it should be moved into its own section in the `others` folder. If no mention of the codename for the game is found, just use an abbreviation for the game's name

**Comment by PTKay** [comment]
This change will be integrated with a direct push to the repository with the suggested fixes. Thanks for your contribution

**Pull request closed without merging**